src/orders/components/OrderGrantRefundPage/components/ShippingInluded.tsx

```diff
@@ -33,7 +33,7 @@ export const ShippingIncluded = ({
         data-test-id="refundShippingCheckbox"
         disabled={!currency || !canRefundShipping}
       >
-        {!currency ? (
+        {!currency || !amount ? (
```

Comment:
what if amount will be 0?

Reply:
I checked, amount (IMoney) is an object, so it should be fine